source/Cosmos.HAL2/TextScreen.cs

```diff
@@ -46,8 +46,16 @@ public TextScreen()
             TextScreenHelpers.Debug("End of TextScreen..ctor");
         }
 
-        public override ushort Rows { get { return 25; } }
-        public override ushort Cols { get { return 80; } }
+        public void UpdateWindowSize()
```

Comment:
check the values of Rows and Cols as anyone can set the values

Reply:
Rows and Cols are public but mFallBackConsole is private thus it cannot be accessed and changed by the user